.env.testnet

```diff
@@ -79,3 +82,24 @@ L2_FORK_RPC_URL=https://optimism-goerli.infura.io/v3/YOUR_INFURA_KEY
 
 # Test network default mnemonic
 TEST_NETWORK_MNEMONIC="test test test test test test test test test test test junk"
+
+
+# *************** SWAP AND BRIDGE ***************
+
+# L1 wdivETH token contract address to be bridged to L2
+L1_TOKEN_ADDR_DIVA=0x91701E62B2DA59224e92C42a970d7901d02C2F24
+
+# L1 wstETH token contract address to be bridged to L2
+L1_TOKEN_ADDR_LIDO=0xB82381A3fBD3FaFA77B3a7bE693342618240067b
+
+# L1 custom bridge for wstETH
+L1_LIDO_BRIDGE_ADDR=0xdDDbC273a81e6BC49c269Af55d007c08c005ea56
+
+# L2 wdivETH token contract
+L2_TOKEN_ADDR_DIVA=0x
```

Comment:
Why is this address empty?
Others are filled in for `testnet`.

Reply:
this address should come from the deployed contract in script 7. Maybe a better way would be to get it from the deployments folder but I thought it would be nice to have it in the env file for the integration tests and also to be in line with what we do with USDT. Once we an an official Sepolia deployment, I can just fill it in

Reply:
Sound good!